diracx-db/src/diracx/db/sql/job/schema.py

```diff
@@ -58,12 +58,12 @@ class Jobs(JobDBBase):
     Owner = Column("Owner", String(64), default="Unknown")
     OwnerGroup = Column("OwnerGroup", String(128), default="Unknown")
     VO = Column("VO", String(32))
-    SubmissionTime = NullColumn("SubmissionTime", DateTime)
-    RescheduleTime = NullColumn("RescheduleTime", DateTime)
-    LastUpdateTime = NullColumn("LastUpdateTime", DateTime)
-    StartExecTime = NullColumn("StartExecTime", DateTime)
-    HeartBeatTime = NullColumn("HeartBeatTime", DateTime)
-    EndExecTime = NullColumn("EndExecTime", DateTime)
+    SubmissionTime = NullColumn("SubmissionTime", DateTime(timezone=True))
+    RescheduleTime = NullColumn("RescheduleTime", DateTime(timezone=True))
+    LastUpdateTime = NullColumn("LastUpdateTime", DateTime(timezone=True))
+    StartExecTime = NullColumn("StartExecTime", DateTime(timezone=True))
+    HeartBeatTime = NullColumn("HeartBeatTime", DateTime(timezone=True))
+    EndExecTime = NullColumn("EndExecTime", DateTime(timezone=True))
```

Comment:
Why? We are anyway always using UTC.

Reply:
Is because we are specifying the timezone in `datetime.now(tz=timezone.utc)`.
While this does not throw any errors in MySQL when the table does not accept a timezone, in PostgreSQL stops the execution due to conflicting formatting.

One way of fixing it is just to completely remove the timezone on the `datetime.now` function, which is not recommended and even [RUFF marks it as a warning](https://docs.astral.sh/ruff/rules/call-datetime-now-without-tzinfo/). 
The other way is by telling the column to take into account the timezone, which in my opinion is the better way of implementing it.

**MySQL:** 
- Changing the table property: 2024-09-26 14:00:32
- Removing `tz=timezone.utc`: 2024-09-26 14:05:53
> No change, both result in the same date format.
	
**PostgreSQL:** 
- Changing the table property: 2024-09-26 13:50:26.213893+00
- Removing `tz=timezone.utc`: 2024-09-26 13:55:57.840623
> Timezone is or is not added at the end.